core/trino-spi/src/main/java/io/trino/spi/security/SystemAccessControl.java

```diff
@@ -137,6 +137,7 @@ default void checkCanViewQueryOwnedBy(SystemSecurityContext context, Identity qu
      * will not be called when the current user is the query owner.
      *
      * @throws AccessDeniedException if not allowed
+     * @deprecated Implement {@link #checkCanViewQueryOwnedBy(SystemSecurityContext, Identity)} instead.
      */
```

Comment:
Add `@Deprecated` annotation. Or should we remove it everywhere?

Reply:
yes, both should be present